KarhooUISDK/CommonUI/LoyaltyInformation/LoyaltyInformationView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LoyaltyInformationView.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 23/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+/// View presenting user-friendly information how many poins shall be added/removed to/from Loyalty.
+struct KarhooLoyaltyInformationView: View {
+
+    private enum Constants {
+        static let iconSide: CGFloat = 16
+    }
+
+    // MARK: - Properties
+
+    let viewModel: ViewModel
+
+    // MARK: - Lifecycle
+
+    init(viewModel: ViewModel) {
+        self.viewModel = viewModel
+    }
+
+    // MARK: - Views
+
+    var body: some View {
+        HStack(spacing: UIConstants.Spacing.small) {
+            Image("kh_uisdk_info_icon", bundle: .current)
+                .resizable()
+                .renderingMode(.template)
+                .foregroundColor(Color(KarhooUI.colors.primary))
+                .frame(width: Constants.iconSide, height: Constants.iconSide)
+            Text(viewModel.text)
+                .font(Font(KarhooUI.fonts.captionBold()))
+                .foregroundColor(Color(KarhooUI.colors.primary))
```

Comment:
Did you created some extension for Color? I cannot find it. Initializer `init Color(uiColor: UIColor) `is available from iOS 15+: https://developer.apple.com/documentation/swiftui/color/init(uicolor:)

Reply:
```
@available(iOS, introduced: 13.0, deprecated: 100000.0, message: "Use Color(uiColor:) when converting a UIColor, or create a standard Color directly")
@available(macOS, unavailable)
@available(tvOS, introduced: 13.0, deprecated: 100000.0, message: "Use Color(uiColor:) when converting a UIColor, or create a standard Color directly")
@available(watchOS, introduced: 6.0, deprecated: 100000.0, message: "Use Color(uiColor:) when converting a UIColor, or create a standard Color directly")
extension Color {

    /// Creates a color from a UIKit color.
    ///
    /// Use this method to create a SwiftUI color from a
    /// <doc://com.apple.documentation/documentation/UIKit/UIColor> instance.
    /// The new color preserves the adaptability of the original.
    /// For example, you can create a rectangle using
    /// <doc://com.apple.documentation/documentation/UIKit/UIColor/3173132-link>
    /// to see how the shade adjusts to match the user's system settings:
    ///
    ///     struct Box: View {
    ///         var body: some View {
    ///             Color(UIColor.link)
    ///                 .frame(width: 200, height: 100)
    ///         }
    ///     }
    ///
    /// The `Box` view defined above automatically changes its
    /// appearance when the user turns on Dark Mode. With the light and dark
    /// appearances placed side by side, you can see the subtle difference
    /// in shades:
    ///
    /// ![A side by side comparison of light and dark appearance screenshots of
    ///   rectangles rendered with the link color. The light variant appears on
    ///   the left, and the dark variant on the right.](Color-init-3)
    ///
    /// > Note: Use this initializer only if you need to convert an existing
    /// <doc://com.apple.documentation/documentation/UIKit/UIColor> to a
    /// SwiftUI color. Otherwise, create a SwiftUI ``Color`` using an
    /// initializer like ``init(_:red:green:blue:opacity:)``, or use a system
    /// color like ``ShapeStyle/blue``.
    ///
    /// - Parameter color: A
    ///   <doc://com.apple.documentation/documentation/UIKit/UIColor> instance
    ///   from which to create a color.
    public init(_ color: UIColor)
}
```